include/drivers/sensor.h

```diff
@@ -80,7 +80,16 @@ enum sensor_channel {
 	/** Ambient temperature in degrees Celsius. */
 	SENSOR_CHAN_AMBIENT_TEMP,
 	/** Pressure in kilopascal. */
-	SENSOR_CHAN_PRESS,
+        SENSOR_CHAN_PRESS,
+        /** Euler heading(yawn) in degrees */
+        SENSOR_CHAN_EULER_H,
+        /** Euler roll in degrees */
+        SENSOR_CHAN_EULER_R,
+        /** Euler pitch in degrees */
+        SENSOR_CHAN_EULER_P,
+        /** Euler heading, roll and pitch in degrees */
+        SENSOR_CHAN_EULER_HRP,
```

Comment:
YRP?
Not sure "yaw" is better than "heading".

Reply:
I choosed heading because bosch also named it like that in the [datasheet](https://www.bosch-sensortec.com/media/boschsensortec/downloads/datasheets/bst-bno055-ds000.pdf) and in the [implementation](https://github.com/BoschSensortec/BNO055_driver/blob/master/bno055.h) too.

Reply:
ok then

Reply:
Pls, correct the other points

Reply:
done